qelem/lib/screens/profile/ChangePassword.dart

```diff
@@ -0,0 +1,139 @@
+                  suffixIcon: IconButton(
+                      icon: Icon(currentHidden
+                          ? Icons.visibility
+                          : Icons.visibility_off),
+                      onPressed: () {
+                        setState(() {
+                          currentHidden = !currentHidden;
+                        });
+                      }),
+                  labelStyle: GoogleFonts.comfortaa(textStyle: textStyle),
+                  border: OutlineInputBorder(
+                      borderRadius: BorderRadius.circular(5.0))),
+            ),
+          ),
+          Padding(
+            padding:
+                const EdgeInsets.symmetric(vertical: 10.0, horizontal: 15.0),
+            child: TextFormField(
+              style: textStyle,
+              obscureText: newHidden,
+              controller: newPasswordController,
+              validator: (value) {
+                if (value!.isEmpty) {
+                  return 'Please enter your new password';
+                }
+              },
+              decoration: InputDecoration(
+                  labelText: 'New Password',
+                  suffixIcon: IconButton(
+                      icon: Icon(
+                          newHidden ? Icons.visibility : Icons.visibility_off),
+                      onPressed: () {
+                        setState(() {
+                          newHidden = !newHidden;
+                        });
+                      }),
+                  labelStyle: GoogleFonts.comfortaa(textStyle: textStyle),
+                  border: OutlineInputBorder(
+                      borderRadius: BorderRadius.circular(5.0))),
+            ),
+          ),
+          Padding(
+            padding:
+                const EdgeInsets.symmetric(vertical: 10.0, horizontal: 15.0),
+            child: TextFormField(
+              style: textStyle,
+              obscureText: confirmHidden,
+              controller: confirmPasswordController,
+              validator: (value) {
+                if (value!.isEmpty) {
+                  return 'Please confirm your current password';
+                } else if (value != newPasswordController.text) {
+                  return 'Passwords don\'t match';
+                }
+              },
+              decoration: InputDecoration(
+                  labelText: 'New Password (again)',
+                  suffixIcon: IconButton(
+                      icon: Icon(confirmHidden
+                          ? Icons.visibility
+                          : Icons.visibility_off),
+                      onPressed: () {
+                        setState(() {
+                          confirmHidden = !confirmHidden;
+                        });
+                      }),
+                  labelStyle: GoogleFonts.comfortaa(textStyle: textStyle),
+                  border: OutlineInputBorder(
+                      borderRadius: BorderRadius.circular(5.0))),
```

Comment:
Remove the border radius, it will be defined in the material app theme. This applies to the other fields too.